app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -116,6 +116,22 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       }
     },
 
+    populateLinkDomain: function (clickData) {
+      var href = clickData.url
+      // We always want mailto links to have an undefined link_domain
+      if (this.isMailToLink(href)) {
```

Comment:
I think this function might be called elsewhere in the code (although I haven't checked in detail) - would it be possible to store the value for this once, rather than calling this function multiple times?

Reply:
@andysellick Yep it does get called previously. I think the code might get a bit more confusing if we try and optimise here though but I'll see how it looks

Reply:
I didn't implement this in the end as I think it'll make the code a bit too confusing - let me know if you want me to explain in more detail.

Reply:
Okay, as long as you've had a look, that's fine.